documentation/modules/adding-source-provider.adoc

```diff
@@ -43,11 +43,11 @@ ifdef::rhv[]
 . Select *Red Hat Virtualization* from the *Type* list.
 . Fill in the following fields:
 
-* *Name*: Name to display in the list of providers
-* *Hostname or IP address*: {manager} host name or IP address
-* *Username*: {manager} user
-* *Password*: {manager} password
-* *CA certificate*: CA certificate of the {manager}. If the {manager} Apache CA certificate was replaced with a third-party CA certificate, both CA certificates need to be specified.
+* *Name*: Name to display in the list of providers.
+* *Hostname or IP address*: {manager} host name or IP address.
+* *Username*: {manager} user.
+* *Password*: {manager} password.
+* *CA certificate*: {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate.
```

Comment:
@RichardHoch 
I would suggest:
"{manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server"
and no periods at the ends of these bullet points.
Also, I think you should change the prerequisites in this section to this:
"CA certificate of the Manager or Apache CA certificate of the Manager if the Manager Apache CA certificate was replaced by a third-party CA certificate"
because now the user only needs one of them, not both.